wcfsetup/install/files/acp/templates/userAdd.tpl

```diff
@@ -5,11 +5,144 @@
+										</a>
+									</li>
+								{/if}
+
+								{if $__wcf->session->getPermission('admin.user.canMailUser')}
+									<li>
+										<a {*
+										*}href="{link controller='UserMail' id=$user->userID}{/link}"{*
+										*}>
+											{lang}wcf.acp.user.action.sendMail{/lang}
+										</a>
+									</li>
+								{/if}
+
+								{if $__wcf->session->getPermission('admin.user.canEditPassword')}
+									<li>
+										<a {*
+										*}href="#" {*
+										*}class="jsSendNewPassword"{*
+										*}>
+											{lang}wcf.acp.user.action.sendNewPassword{/lang}
+										</a>
+									</li>
+								{/if}
+							{/if}
+
+							{if $__wcf->session->getPermission('admin.user.canExportGdprData')}
+								<li
+									<a {*
+										*}href="{link controller='UserExportGdpr' id=$user->userID}{/link}"{*
+										*}>
+										{lang}wcf.acp.user.exportGdpr{/lang}
+									</a>
+								</li>
+							{/if}
+
+							{if $__wcf->session->getPermission('admin.user.canDeleteUser')}
+								<li class="dropdownDivider"></li>
+								<li><a href="#" class="jsDispatchDelete">{lang}wcf.global.button.delete{/lang}</a></li>
+								<li><a href="#" class="jsDeleteContent">{lang}wcf.acp.content.removeContent{/lang}</a></li>
+							{/if}
+						</ul>
+					</div>
+				</li>
+				<script data-relocate="true">
+					require(['Language',
+							 'WoltLabSuite/Core/Acp/Ui/User/Editor',
+							 'WoltLabSuite/Core/Acp/Ui/User/Content/Remove/Handler',
+							 'WoltLabSuite/Core/Acp/Ui/User/Action/DisableAction',
+							 'WoltLabSuite/Core/Acp/Ui/User/Action/SendNewPasswordAction',
+							 'WoltLabSuite/Core/Acp/Ui/User/Action/ToggleConfirmEmailAction',
+							 'WoltLabSuite/Core/Controller/Clipboard',
+						],
+						function (
+							 Language,
+							 AcpUiUserList,
+							 AcpUserContentRemoveHandler,
+							 DisableAction,
+							 SendNewPasswordAction,
+							 ToggleConfirmEmailAction,
+							 Clipboard
+						) {
+							Language.addObject({
+								'wcf.acp.user.sendNewPassword.workerTitle': '{jslang}wcf.acp.user.sendNewPassword.workerTitle{/jslang}',
+								'wcf.acp.user.action.sendNewPassword.confirmMessage': '{jslang}wcf.acp.user.action.sendNewPassword.confirmMessage{/jslang}',
+								'wcf.acp.worker.abort.confirmMessage': '{jslang}wcf.acp.worker.abort.confirmMessage{/jslang}',
+								'wcf.acp.content.removeContent': '{jslang}wcf.acp.content.removeContent{/jslang}',
+							});
+
+							const dropdownElement = document.querySelector("#userListDropdown{@$user->userID}");
+
+							const deleteContent = document.querySelector(".jsDeleteContent");
+							if (deleteContent !== null) {
+								new AcpUserContentRemoveHandler(deleteContent, {@$user->userID});
+							}
+
+							const sendNewPassword = document.querySelector(".jsSendNewPassword");
+							if (sendNewPassword !== null) {
+								new SendNewPasswordAction.default(sendNewPassword, {@$user->userID}, dropdownElement);
```

Comment:
Import as `{ SendNewPasswordAction }` instead to avoid the `.default` (same below).